main/zoe/api/zoe-contract-facet.md

```diff
@@ -313,9 +321,14 @@ Returns the `amountMath` object associated with the `brand` argument.
 ```js
 const assetAmountMath = zcf.getAmountMath(assetAmount.brand);
 ```
-## zcf.shutdown()
+## zcf.stopAcceptingOffers()
+- The contract requests Zoe to not accept offers for this contract instance. **tyg todo: No way of 
+starting it back up and accepting again?** It can't be called from outside the contract 
+unless the contract explicitly makes it accessible.
```

Comment:
This hasn't been added yet.  I suppose I should add this now.

Reply:
Let's just table this particular documentation change into a later PR rather than change your PR, @Chris-Hibbert

Reply:
Nah, it's okay to include this. I've prepared a [PR that adds `stopAcceptingOffers()`](https://github.com/Agoric/agoric-sdk/pull/1841). 

Since Dean has requested that we coordinate before releasing these changes, we might as well have all the related changes ready at the same time.  (And Dean is the one who requested `stopAcceptingOffers()`.)